pyseir/inference/model_fitter.py

```diff
@@ -624,7 +637,7 @@ def fit(self):
                 eps_error=self.fit_results[f"{epsilon}_error"],
                 lower_bound_reff=ModelFitter.REFF_LOWER_BOUND,
             )
-            # TODO: Add structured logging if this change is significant
+            log.info(f"epsilon: {self.fit_results[epsilon]} adjusted: {adjusted_epsilon}")
```

Comment:
Think of the struct log as an object that you can add key-value pairs to like : `log.info("Adjusted Epsilon", original=self.fit_results[epsilon], adjusted=adjusted_epsilon)`

Reply:
And throw fips in there if it's in scope.

Reply:
Ok, also feel free to change this if you care, we are about to remove this adjustment anyway.